template/examplesource/ExampleProject/BuildXboxOneGDK.bat.ejs

```diff
@@ -14,7 +14,7 @@ rem # Full Path. In theory it should not be edited (the engine path/version are
 set uatPath=%uePath%\UE_%ueVersion%\Engine\Build\BatchFiles\RunUAT.bat
 
 rem # Target Platform
-set targetPlatform=XboxOneGDK
+set targetPlatform=<% if (ueTargetVersionMajor == 5 && ueTargetVersionMinor >= 4) { %>XB1<% } else { %>XboxOneGDK<% } %>
```

Comment:
Would it make sense to use >= for the major version as well? Probably won't matter in the long run, but there's a chance it avoids a future unexpected bug.

Reply:
I guess it's slightly more complicated than that. Maybe if (ueTargetVersionMajor == 5 && ueTargetVersionMinor >= 4 || ueTargetVersionMajor > 5)